### PR TITLE
Common java8

### PR DESCRIPTION
**Pull request by anaHue**

### CODE REVIEW

File: $common/run [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/python3
```
Comment (anthonygego) [inline $common/run]: I guess this file shouldn't be present here.

File: m01Q11/run [hunk 2]
```diff
@@ -1,40 +0,0 @@
-#!/bin/python3
```
Comment (anthonygego) [inline m01Q11/run]: Should this be deleted ?

File: $common/run (hunk 1 above)
Comment (anaHue) [reply $common/run]: Indeed I changed that now.

File: m01Q11/run (hunk 2 above)
Comment (anaHue) [reply m01Q11/run]: Indeed I changed that now.

**Pull request merged**